sites-available/README.md

```diff
@@ -9,4 +9,9 @@ example.com (handles traffic from both www.example.com and example.com)
 foobar.com (as above)
 test.foobar.com (handles traffic from both www.test.foobar.com and test.foobar.com)
 ```
+The HTTPS no-default server config requires a valid certificate (self signed will suffice).
+One can generate the certificate with: 
+```
+sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/nginx/ssl/nginx.key -out /etc/nginx/ssl/nginx.crt
```

Comment:
@leonklingele To be honest I did not check very deep on type of cert it will be generated as I just needed something to work and since the 444 result was given I suspected not much focus is needed on that part. But if you have a better config, I'm more than happy to use it.